### ISSUE TITLE
Pd prototype vs [netreceive] vs Windows

### ISSUE THREAD

**Issue by jamshark70** [VCVRack/VCV-Prototype#51]
I haven't been able to get [netreceive] to work in a Pd patch loaded into VCV Prototype in Windows. It works fine in Linux.

Simple test Pd patch:

![pd-simple-rack-test](https://user-images.githubusercontent.com/318301/119922470-6f6db500-bfa2-11eb-91e0-8b2b702b48ee.png)

In Rack, I create a Prototype module and load this patch into it.

Then, in SuperCollider, I do:

```supercollider
n = NetAddr("127.0.0.1", 7878);
n.sendMsg(\test, 123);
```

- Linux: The Prototype module happily displays "received message test 123." (Same if I use a broadcast address: `n = NetAddr("255.255.255.255", 7878); NetAddr.broadcastFlag = true;` and send a message -- of course I'm changing the message contents every time, to see that the display changed.)

- Windows: Nothing happens. If I have the Pd patch open in Pure Data as well as the Prototype module, Pd prints the message in its console (whether the Windows machine originates the message, or another machine on the LAN, both work). If I add a cvOSCcv module to the Rack patch and send '/ch/1' messages to it (again, from localhost *or* a remote machine), then the output voltage changes -- so I know that Windows Firewall is not blocking messages into Rack. But the Prototype module fails to respond.

I do see a [comment](https://github.com/VCVRack/VCV-Prototype/issues/40#issuecomment-663053289) "I'm not sure where pure data uses the Winsock library. I guess for OSC and other networking?" So it may be that Pd networking has been accidentally disabled (but for Windows only)...?

**Comment by mxa** [comment]
Could you quickly confirm that it works on Windows when opening the patch in Vanilla Pd?

**Comment by mxa** [comment]
> I do see a [comment](https://github.com/VCVRack/VCV-Prototype/issues/40#issuecomment-663053289) "I'm not sure where pure data uses the Winsock library. I guess for OSC and other networking?" So it may be that Pd networking has been accidentally disabled (but for Windows only)...?

I don't think that this has anything to do with it since the Winsock flag is set in the build.
https://github.com/VCVRack/VCV-Prototype/blob/6980f0f0aca2517ea4a44e749515fe868d4ff532/Makefile#L221

**Comment by jamshark70** [comment]
> Could you quickly confirm that it works on Windows when opening the patch in Vanilla Pd?

Information already provided: "If I have the Pd patch open in Pure Data as well as the Prototype module, Pd prints the message in its console." (Of course I tested this -- I had to be sure that it wasn't a case of blocked network messages.) It is definitively: Pd responds, VCV doesn't.
